RELEASE-NOTES.txt

```diff
@@ -1,12 +1,12 @@
-Welcome to the 0.9.5 open source release of the Simulation Engine! SimEng is a fast, easy to use and modify, cycle-level simulator for CPUs. Its initial focus is on simulating single Arm cores in server CPUs, and so the instruction set architecture (ISA) target is initially Arm9.2-a+SVE+SME. RISC-V has also seen some inital support, starting with rv64ima. Later versions of the Arm ISA, additional RISC-V ISA extensions and other ISAs, will be supported in future releases.
+Welcome to the 0.9.6 open source release of the Simulation Engine! SimEng is a fast, easy to use and modify, cycle-level simulator for CPUs. Its initial focus is on simulating single Arm cores in server CPUs, and so the instruction set architecture (ISA) target is initially Arm9.2-a+SVE+SME. RISC-V has also seen some inital support, starting with rv64ima. Later versions of the Arm ISA, additional RISC-V ISA extensions and other ISAs, will be supported in future releases.
```

Comment:
This line needs updating RE RISC-V. Currently supports rv64imafd and c hopefully

Reply:
Changed to rv64imafd for now

Reply:
I'd maybe remove "initial support" now as well as this is a typical set of extensions and it is a bit more filled out than before. ~"There is also RISC-V support covering rv64imafd"

Reply:
Will make that change

Reply:
updated to include the c extension where appropriate